src/components/standalone/openvpn_rw/RWAccountsManager.vue

```diff
@@ -26,6 +26,7 @@ import DeleteRWAccountModal from './DeleteRWAccountModal.vue'
 import CreateOrEditRWAccountDrawer from './CreateOrEditRWAccountDrawer.vue'
 import RenewCertificateDrawer from './RenewCertificateDrawer.vue'
 import { useNotificationsStore } from '@/stores/notifications'
+import { downloadFile, deleteFile } from '@/lib/standalone/fileUpload'
```

Comment:
It appears it's not imported FontAwesomeIcon, it however works because it's loaded somewhere else

```suggestion
import { downloadFile, deleteFile } from '@/lib/standalone/fileUpload'
import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
```